packages/api/src/clickhouse/__tests__/clickhouse.test.ts

```diff
@@ -1,4 +1,14 @@
+import _ from 'lodash';
```

Comment:
Great tests!! not required in this PR, but I'd write tests for `buildEventSeriesQuery` and `queryMultiSeriesChart` to make sure queries are properly generated

Reply:
Ty - I think testing those fns are pretty low ROI test-wise compared to testing it all e2e to make sure the values are actually correct, otherwise I think those tests alone are just going to test fragile implementation details.